### DISCUSSION TITLE
Laisser la possibilité de créer un compte pour sauvegarder ses données

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2637] [Idées]
Retour utilisateur :

> ce serait super de pouvoir se créer un compte et d'enregistrer et mettre à jour ses données afin de pouvoir faire évoluer les projections sans tout renseigner ;-)

**Comment by laem** [comment]
Une piste pour éviter d'avoir à gérer une base de donnée sensible :) 

https://0data.app/fr/

**Comment by mquandalle** [comment]
Pour l'authentification, plutôt que d'avoir à recréer un identifier email + mot de passe, une piste possible est de passer par le compte portailpro.gouv qui permettra dès l'année prochaine de s'identifier sur impots.gouv.fr, urssaf.fr et le site des douanes.

**Comment by mquandalle** [comment]
Sur le même sujet :

- https://github.com/datagir/nosgestesclimat-site/issues/400
